kalkulator.py

```diff
@@ -1,11 +1,15 @@
 def get_info():
-print("Witaj to jest prosty kalkulator!")
+    print("Witaj to jest prosty kalkulator!")
 def dodaj(a, b):
     wynik = a + b
     return wynik
+
+def odejmij(a, b):
```

Comment:
Moim zdaniem tutaj lepiej rozbić to na dwie linijki wprowadzając zmienną "wynik"